crypto/objects/objects.txt

```diff
@@ -1121,6 +1121,24 @@ sigAlgs 13	: id-rsassa-pkcs1-v1_5-with-sha3-224	: RSA-SHA3-224
 sigAlgs 14	: id-rsassa-pkcs1-v1_5-with-sha3-256	: RSA-SHA3-256
 sigAlgs 15	: id-rsassa-pkcs1-v1_5-with-sha3-384	: RSA-SHA3-384
 sigAlgs 16	: id-rsassa-pkcs1-v1_5-with-sha3-512	: RSA-SHA3-512
+sigAlgs 17      : id-ml-dsa-44      : ML-DSA-44
+sigAlgs 18      : id-ml-dsa-65      : ML-DSA-65
+sigAlgs 19      : id-ml-dsa-87      : ML-DSA-87
+sigAlgs 20      : id-slh-dsa-sha2-128s	: SLH-DSA-SHA2-128S
+sigAlgs 21      : id-slh-dsa-sha2-128f	: SLH-DSA-SHA2-128F
+sigAlgs 22      : id-slh-dsa-sha2-192s	: SLH-DSA-SHA2-192S
+sigAlgs 23      : id-slh-dsa-sha2-192f	: SLH-DSA-SHA2-192F
+sigAlgs 24      : id-slh-dsa-sha2-256s	: SLH-DSA-SHA2-256S
+sigAlgs 25      : id-slh-dsa-sha2-256f	: SLH-DSA-SHA2-256F
+sigAlgs 26      : id-slh-dsa-shake-128s	: SLH-DSA-SHAKE-128S
+sigAlgs 27      : id-slh-dsa-shake-128f	: SLH-DSA-SHAKE-128F
+sigAlgs 28      : id-slh-dsa-shake-192s	: SLH-DSA-SHAKE-192S
+sigAlgs 29      : id-slh-dsa-shake-192f	: SLH-DSA-SHAKE-192F
+sigAlgs 30      : id-slh-dsa-shake-256s	: SLH-DSA-SHAKE-256S
+sigAlgs 31      : id-slh-dsa-shake-256f	: SLH-DSA-SHAKE-256F
+sigAlgs 32      : id-hash-ml-dsa-44-with-sha512 : HASH-ML-DSA-44-WITH-SHA512
+sigAlgs 33      : id-hash-ml-dsa-65-with-sha512 : HASH-ML-DSA-65-WITH-SHA512
+sigAlgs 34      : id-hash-ml-dsa-87-with-sha512 : HASH-ML-DSA-87-WITH-SHA512
```

Comment:
Any particular reason why we are not adding the pre-hash SLH-DSA variants while we are making this update anyway? (Approved anyway regardless of answer)

Reply:
An oversight, though not immediately needed, it makes sense to add these in this PR. Sorry about that. Added.